### DISCUSSION TITLE
Drive indicator

### DISCUSSION THREAD

**Discussion by pongloongyeat** [elementary/wingpanel#387] [Ideas]
There's no way of knowing when someone has plugged in their removable USB drive unless they open Files and see the drive added on the sidebar. Since elementary's HIG reserves indicators for system statuses, adding a drive indicator would probably work here and shows only when a drive is connected.

Quick mockup (it just uses Granite.SourceList):

![image](https://user-images.githubusercontent.com/31680656/111021982-e74c2a80-840a-11eb-97a8-1d87ad922f55.png)

**Comment by teamcons** [comment]
I like this one.
